IntegrationWithPharmacies/MedicineInformationApi/Service/MedicineInformationService.cs

```diff
@@ -0,0 +1,109 @@
+﻿using MedicineInformationApi.Adapter;
+using MedicineInformationApi.DbContextModel;
+using MedicineInformationApi.Dto;
+using MedicineInformationApi.Model;
+using MedicineInformationApi.Repository;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace MedicineInformationApi.Service
+{
+    public class MedicineInformationService : IMedicineInformationService
+    {
+        public MedicineInformationRepository MedicineWithQuantityRepository { get; }
+        public IMedicineInformationRepository IMedicineWithQuantityRepository { get; }
```

Comment:
Change name of interface IMedicineWithQuantityRepository to _medicineWithQuantityRepository.